integrations/extensions/starter-kits/hubspot/readme.md

```diff
@@ -20,7 +20,21 @@ HubSpot has a fantastic public API documentation that is very custom extensions
 
 The `basic` kit implements a single yet useful API call for retrieving contact information. If you are new to custom extensions, we recommend you start there to get an idea on how to use custom extensions with skills. The `advanced` folder contains a kit with more advanced functionality, which you can use as a creative springboard for complex use cases.
 
-## Pre-Req 1: Getting Auth Keys
+## Pre-Req 1: Getting Private Apps Access Token
+#### ** Updated Aug. 29, 2022 **
+
+Hubspot has increased their level of security for making API calls. They no longer support one API Key but rather support keys per Application. Hubspot now support two types of Apps: Private and Public. Public Apps require OAuth authentication which currently IBM Watson Assistant does not support. For the time being, we recommend using Private Apps which are based on Bearer Authentication. Please note, to follow Hubspot guidelines on sharing this Bearer token.
+
+To call the HubSpot API, you will need an Private App access token. To obtain this key:
+
+1. Click on `Settings` (the cog wheel icon) at the top right of the page, then click `Integrations` in the left menu to open a drop-down, and select `Private Apps`.
+
+1. If you do not already have one, `Create a private app`. Enter your app's name than, click the `Scopes` tab select the scopes `tickets` and `crm.objects.contacts.read`. Click `Create app` in the top right. Copy the access token and save for future steps.
```

Comment:
I think that for instance to create a ticket you need additional scopes

Reply:
Hi @amiddavid I have tested with these scopes and created a ticket over WA. it worked!